packages/graphql-tool-utilities/ast.ts

```diff
@@ -21,66 +19,57 @@ export enum OperationType {
 
 export interface Variable {
   name: string;
-  type?: GraphQLInputType;
-}
-
-export interface TypedVariable {
-  name: string;
-  type: GraphQLInputType;
+  type?: GraphQLType;
```

Comment:
Hmm this change seems incorrect? Variables can only be input types

Reply:
it's as per the internal types, i can try and change though to see if it makes a big difference. I'm guessing that will break compat with internal types though.

Reply:
I was thinking of the output type downcast, this one seems fine 👍